docs/obey_outcomes.rst

```diff
@@ -0,0 +1,30 @@
+=============
+Obey Outcomes
+=============
+
+By default, when using ``ignore_outcome`` in your Tox configuration,
+any build errors will show as successful on Travis. This might not
+be desired, as you might want to control allowed failures inside your
+``.travis.yml``. To cater this need, you can set ``obey_outcomes`` to
+``False``. This will override ``ignore_outcome`` by setting it to
+``False`` for all environments.
+
+
+Usage
+=====
+
+Configure the allowed failures in the build matrix in your ``travis.yml``:
+
+.. code-block:: yaml
+
+    matrix:
+      allow_failures:
+      - python: 3.6
+        env: DJANGO=master
+
+And in your ``tox.ini``:
+
+.. code-block:: ini
+
+    [travis]
+    obey_outcomes = False
```

Comment:
I'm seeing what @ryanhiebert is saying here about the name confusion. I would have expected `obey_outcomes` to use a true value, that you're obeying the original outcome and in effect are disabling the `ignore_outcome` setting. Your interpretation here is that the setting is now not obeying the outcome after `ignore_outcome` has been applied. 

Maybe `unignore_outcomes` is the better name? The setting would then expect a true value.